packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -647,9 +669,12 @@ export class InputManager {
                     }
                     // at least one double click is required to be check and exclusive double click is enabled
                     else {
-                        // wait that no double click has been raised during the double click delay
-                        this._previousDelayedSimpleClickTimeout = this._delayedSimpleClickTimeout;
-                        this._delayedSimpleClickTimeout = window.setTimeout(this._delayedSimpleClick.bind(this, btn, clickInfo, cb), InputManager.DoubleClickDelay);
+                        // Queue up a delayed click, just in case this isn't a double click
+                        const delayedClick = new _ClickEvent();
+                        delayedClick.evt = evt;
+                        delayedClick.clickInfo = clickInfo;
+                        delayedClick.timeoutId = window.setTimeout(this._delayedSimpleClick.bind(this, btn, clickInfo, cb), InputManager.DoubleClickDelay);

```

Comment:
we should explain that this will probably not be considered a user interaction. It's a small issue, but one won't be able to use this event to execute actions that require user interaction (like, for example, entering webxr or enabling audio).

Reply:
Added comment to more properly clarify usage.